$NetBSD: patch-Makefile,v 1.1 2019/05/28 20:43:04 maya Exp $

 - Remove MACOSX_DEPLOYMENT_TARGET as it will give problem on newer macOS
     and also targeting/building for current macOS version is enough

--- Makefile.orig	2017-09-15 23:07:33.000000000 +0000
+++ Makefile
@@ -12,7 +12,7 @@ OBJS= src/$T.o
 lib: src/lfs.so
 
 src/lfs.so: $(OBJS)
-	MACOSX_DEPLOYMENT_TARGET="10.3"; export MACOSX_DEPLOYMENT_TARGET; $(CC) $(LIB_OPTION) -o src/lfs.so $(OBJS)
+	$(CC) $(LIB_OPTION) -o src/lfs.so $(OBJS)
 
 test: lib
 	LUA_CPATH=./src/?.so lua tests/test.lua
